### PR TITLE
Update quote of day to libingester 2

### PR DESCRIPTION
**Pull request by syndacks**

### CODE REVIEW

File: examples/quote_of_day.js [hunk 1]
```diff
@@ -1,72 +1,81 @@
+
```
Comment (sgnn7) [inline examples/quote_of_day.js]: ?

File: examples/quote_of_day.js [hunk 2]
```diff
@@ -1,72 +1,81 @@
...
             const uri = $pages(this).attr('href');
             return url.resolve(HOMEPAGE, uri);
         }).get();
 
-        Promise.all(articles_links.map((uri) => ingestArticleProfile(hatch, uri)))
-        .catch(err => console.log(err.stack)).then(() => {
+        Promise.all(articles_links.map((uri) => ingest_article(hatch, uri))).then(() => {
             return hatch.finish();
-        });
+          }).catch((err) => console.log(err));
```
Comment (sgnn7) [inline examples/quote_of_day.js]: You can just change this to: `}).catch(console.log);`

**Comment by syndacks** [comment]
Thanks @sgnn7 , changes made.

File: examples/quote_of_day.js [hunk 3]
```diff
@@ -1,71 +1,78 @@
...
             const uri = $pages(this).attr('href');
             return url.resolve(HOMEPAGE, uri);
         }).get();
 
-        Promise.all(articles_links.map((uri) => ingestArticleProfile(hatch, uri)))
-        .catch(err => console.log(err.stack)).then(() => {
+        Promise.all(articles_links.map((uri) => ingest_article(hatch, uri))).then(() => {
             return hatch.finish();
-        });
+          }).catch(console.log);
```
Comment (sgnn7) [inline examples/quote_of_day.js]: Wrong indentation

File: examples/quote_of_day.js [hunk 4]
```diff
@@ -1,71 +1,78 @@
...
 }
 
 function main() {
     const hatch = new libingester.Hatch('quote_of_day', 'en');
-    // make request to the index (home) page
     libingester.util.fetch_html(HOMEPAGE).then(($pages) => {
       // retrieve article URLs; '-2n+2' returns ~30 articles instead of 2,000+
-        const articles_links = $pages('#Calendar div:nth-child(-2n+2) a').map(function() {
+      //                        '-n+28' returns ~901 articles
+          const articles_links = $pages('#Calendar div:nth-child(-n + 28) a').map(function() {
```
Comment (sgnn7) [inline examples/quote_of_day.js]: Wrong indentation

**Comment by sgnn7** [comment]
CR Pass

**Pull request merged**